Recruit-Training/Embedded-Recruit-Training/Core/Src/BlueLedToggleTask.c

```diff
@@ -7,11 +7,40 @@ void blueLedToggleTask(void const* arg)
 
     osMutexId_t* canMutex = (osMutexId_t*)arg; // Get mutex that was passed as an argument
 
+    static const uint32_t BLUE_LED_STATUS_STDID = 0xCCC;
+    static const uint32_t BLUE_LED_TOGGLE_FREQ = 1000;
+
+
     for (;;)
     {
         //TODO: Add BLUE_LED_TOGGLE_FREQ to prevWakeTime
+        prevWakeTime += BLUE_LED_TOGGLE_FREQ;
         osDelayUntil(prevWakeTime);
+
         //TODO: Check blue toggle flag and toggle blue LED
+        if(blueLedToggleFlag) 
+        {
+            HAL_GPIO_TogglePin(GPIOA, LED_RED_Pin);
+        }
+
         //TODO: Send CAN message indicating current state of LED
+        uint8_t BLUE_LED_STATUS_ID = HAL_GPIO_ReadPin(GPIOA, LED_RED_Pin);
```

Comment:
Same thing here. Should be camelCase and use the appropriate variable for `GPIOA`